libraries/Microsoft.Bot.Builder/Adapters/TestFlow.cs

```diff
@@ -61,7 +61,7 @@ public TestFlow(Task testTask, TestFlow flow)
         /// <param name="adapter">The test adapter to use.</param>
         /// <param name="bot">The bot containing the turn processing logic to test.</param>
         public TestFlow(TestAdapter adapter, IBot bot)
-            : this(adapter, (ctx) => bot.OnTurnAsync(ctx))
+            : this(adapter, (ctx, cancellationToken) => bot.OnTurnAsync(ctx, cancellationToken))
```

Comment:
Technically there's no need to create an explicit closure here. If you just inline `bot.OnTurnAsync` the compiler will implicitly create a delegate around the instance method for you and you avoid an extra jump through the closure trampoline.